pkg/packagekit/updates.jsx

```diff
@@ -1080,19 +1080,24 @@ class OsUpdates extends React.Component {
                     this.setState({ checkRestartAvailable: true, checkRestartRunning: false, restartPackages });
                 })
                 .catch((exception, data) => {
-                    // common cases: this platform does not have tracer installed
-                    if (!exception.message?.includes("ModuleNotFoundError") &&
+                    // tracer not installed? then fall back to dnf needs-restarting
+                    if (exception.message?.includes("ModuleNotFoundError") ||
                         // or supported (like on Arch)
```

Comment:
This comment is a continuation of the now deleted "this platform does not have tracer installed" ... "or supported".  Now it just hangs here alone and needs updating.

Reply:
Hmm, so we fall back to dnf when tracer is installed but reports UnsupportedDistribution? That makes sense, I guess, but out of interest, can it really happen?

Reply:
It *probably* can't happen, as tracer doesn't exist at all any more in RHEL 10. However, I don't want to make unnecessary assumptions. Tracer could come from an old package, a COPR, or even places like pypi?

On Arch we indeed do two failing checks (neither tracer nor dnf exists), but these should be fast enough? This arguably isn't super efficient, but it also doesn't run very often.

The verrutschte comment is sad now indeed, I'll fix it in a follow-up.

Reply:
> On Arch we indeed do two failing checks (neither tracer nor dnf exists), but these should be fast enough? This arguably isn't super efficient, but it also doesn't run very often.

Yes, I agree.

Reply:
#21785